dpgen2/entrypoint/args.py

```diff
@@ -17,6 +17,29 @@
 from dpgen2.exploration.report import conv_styles
 
 
+def dp_dist_train_args():
+    doc_config = "Configuration of training"
+    doc_template_script = "File names of the template training script. It can be a `List[Dict]`, the length of which is the same as `numb_models`. Each template script in the list is used to train a model. Can be a `Dict`, the models share the same template training script. "
```

Comment:
```suggestion
    doc_template_script = "File names of the template training script. It can be a `List[Dict]`, the length of which is the same as `numb_models`. Each template script in the list is used to train a model. Can be a `str`, the models share the same template training script. "
```